extensions/wikipedia/package.json

```diff
@@ -40,9 +41,44 @@
     "Web",
     "News"
   ],
+  "preferences": [
+    {
+      "name": "zhVariant",
+      "title": "Chinese Language Variants",
+      "description": "Select the default language variant for the Chinese Wikipedia",
+      "required": false,
+      "type": "dropdown",
+      "data": [
+        {
+          "title": "臺灣正體",
+          "value": "zh-tw"
+        },
+        {
+          "title": "大陆简体",
+          "value": "zh-cn"
+        },
+        {
+          "title": "香港繁體",
```

Comment:
At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.